convene-web/Gemfile

```diff
@@ -10,14 +10,13 @@ ruby '2.7.1'
 gem 'dotenv-rails', groups: [:development, :test]
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 6'
+gem 'rails', '~> 6.1'
```

Comment:
I think we can keep `gem 'rails', '~> 6'` and just run `bundle update rails` to achieve the same result and avoid locking to `6.1`.

Reply:
What does that buy us?